problem_builder/templates/html/swipeblock.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+{% if not hide_header %}
+<h4 class="question-title" id="heading_{{ self.html_id }}">{{ self.display_name_with_default }}</h4>
+{% endif %}
+<img src={{ self.expand_static_url(self.img_url) }}
```

Comment:
This does not work; I get "Error: Variables and attributes may not begin with underscores: 'self._expand_static_url'"

Reply:
Also missing end of `<img` tag.

Reply:
You will also have to define an `alt` attribute: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/accessibility.html#alt-attributes